src/types.d.ts

```diff
@@ -1,15 +1,32 @@
-export enum GENDER {
-  MALE,
-  FEMALE,
-  APACHE_HELICOPTER,
+export type Gender = "Male" | "Female";
+export type Size = "small" | "medium" | "large";
+export type Email = string | null;
+export type PhoneNumber = number | null;
+export type Text = string;
+export type Direction = "shrink" | "grow";
+
+export interface PropsSingleData {
+  first_name: string;
+  last_name: string;
+  gender: string;
+  image: URL;
+  occupation: string;
+  about: string;
+  card_size: string;
+  contact: string;
 }
 
 export interface SingleData {
```

Comment:
mozna nazwać interfejs np. `CardProps` bo w tym momencie po kilku miesiącach od dodania zadania ja już nie pamiętam do czego to miało służyć - a w ten sposób już po samej nazwie byłoby to wiadome